packages/core/test/scxml.test.ts

```diff
@@ -59,7 +54,7 @@ const testGroups = {
     'history1',
     'history2',
     'history3',
-    // 'history4', // TODO: support history nodes on parallel states
+    'history4', // TODO: support history nodes on parallel states
```

Comment:
this got uncommented, but has a TODO comment now, is this test passing now? if yes - why there is a TODO besides it?